packages/evolution-frontend/src/components/pages/SurveyParticipant.tsx

```diff
@@ -101,7 +101,7 @@ const SurveyParticipant: React.FC = () => {
 
     // FIXME See if we can use react Suspense instead of this logic for the loading page (https://react.dev/reference/react/Suspense)
     if (!interviewLoaded || !interview || !interview.sectionLoaded) {
-        surveyHelperNew.devLog('%c rendering empty survey', 'background: rgba(0,0,0,0.1);');
+        surveyHelperNew.devLog('%c rendering empty survey ', 'background: rgba(0,0,0,0.1);');
```

Comment:
Il y a un espace qui a été ajouté. D'ailleurs je déteste ce message, ça veut rien dire! J'aimerais mieux voir "interview not [yet] available or not loaded"